echopype/convert/api.py

```diff
@@ -154,6 +154,7 @@ def to_file(
         source_file=echodata.source_file,
         file_format=format_mapping[engine],
         save_path=save_path,
+        output_storage_options=output_storage_options,
```

Comment:
There's the culprit! It's trying to access AWS instead of local minio!

Reply:
Works locally because you're actually accessing AWS. :stuck_out_tongue_closed_eyes: 
![Screenshot from 2021-04-26 17-22-49](https://user-images.githubusercontent.com/17802172/116166953-0dc0df00-a6b4-11eb-8bbd-a4e6d59982ba.png)